roles/create-openshift-resources/tasks/promote_image.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
In order for promotion to be successful, the user needs to have the proper permissions in OpenShift. We should create another ansible role / tasks to do this. See the docs for more details:
https://docs.openshift.com/enterprise/3.2/install_config/install/docker_registry.html#access-user-prerequisites